Task_DEV_5/Task_DEV_5/SecondSorter.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+using System.Text;
+using System.IO;
+
+namespace task_DEV_5
+{
+    class SecondTextSort
```

Comment:
Bad class name. It is not understandable from the context what means FirstSorter and SecondSorter.

Reply:
Instead of `Sort` better to use `Sorter` in class name.